counsel.el

```diff
@@ -5145,6 +5145,20 @@ in the current window."
             :unwind #'counsel--switch-buffer-unwind
             :update-fn 'counsel--switch-buffer-update-fn))
 
+;;;###autoload
+(defun counsel-switch-buffer-other-window ()
+  "Switch to another buffer in another window.
+Display a preview of the selected ivy completion candidate buffer
+in the current window."
+  (interactive)
+  (ivy-read "Switch to buffer in another window: " 'internal-complete-buffer
```

Comment:
Nitpick: "Switch to buffer in **other** window", for consistency with `switch-to-buffer-other-window`.